tests/Controller.spec.ts

```diff
@@ -765,7 +765,7 @@ describe('Controller spec', () => {
             )
           ).value.ok.ok,
           {
-            collateral: 90 - (50 * 90) / 100,
+            collateral: 90,
```

Comment:
This fix should be a code optimization, not a feature change, but do you see what the difference is here?

Reply:
Yeah, there was an logic error in controller/utils.rs `get_hypothetical_account_liquidity` function